### PR TITLE
git-standup 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by md5**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

**Comment by md5** [comment]
This formula does not currently pass `brew audit` because [the source repository](https://github.com/kamranahmedse/git-standup) is newer than 30 days. However, it already has 800+ stars, so I thought I'd open a PR anyways.

**Comment by ilovezfs** [comment]
This will need `bottle :unneeded`

**Comment by md5** [comment]
@ilovezfs Fixed and amended

**Comment by apjanke** [comment]
Test is failing.

```
==> git standup --help
No manual entry for git-standup
Error: git-standup: failed
Failed executing: git standup --help
```

This project is an 80 line script and doesn't do `--help` or `--version`. Despite the number of stars and forks, that 30 day waiting period might give it time to mature a bit. In order to test it as is, you need to either create a test `git` repo to run it against, or call it with an intentionally-invalid command line containing too many options and check its [stderr output](https://github.com/kamranahmedse/git-standup/blob/4f831c6ad3a919e369dc83e1dc9ddd9f7e87ef03/git-standup#L3-L6).

File: Formula/git-standup.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  desc "Git extension to generate reports for standup meetings"
+  homepage "https://github.com/kamranahmedse/git-standup"
+  url "https://github.com/kamranahmedse/git-standup/archive/1.0.0.tar.gz"
+  sha256 "fd3808fb8414308413248b2f47646f51ce7bf7a6b59beb257fbde2e983b60127"
+  head "https://github.com/kamranahmedse/git-standup.git"
+
+  bottle :unneeded
+
+  def install
+    bin.install "git-standup"
```
Comment (apjanke) [inline Formula/git-standup.rb]: The project provides a `make install` target. Should probably just use that instead, even if this is all that it's doing right now. That way the formula won't have to change if the project adds other things, like a man page or additional commands.

**Comment by md5** [comment]
@apjanke the `--help` command for `git` is handled by delegating to `man`. We're discussing the addition of a `man` page at https://github.com/kamranahmedse/git-standup/issues/9 to correct this.

Regarding adding a more robust test that involves creating a `.git` repo and checking the output of the command, that's also a good idea, although more involved.

File: Formula/git-standup.rb (hunk 1 above)
Comment (md5) [inline Formula/git-standup.rb]: I noticed that. The reason I didn't use the `make install` target is that it does not allow for `prefix` relocation. I'll raise that topic in the discussion at https://github.com/kamranahmedse/git-standup/issues/9

Should I close this PR in the meantime?

**Comment by apjanke** [comment]
Cool.

Since this project is still by Homebrew's Acceptable Formulae criteria, and it looks like it's going through active development, I'm going to recommend that you or the `github-standup` developers [create your own Tap for it](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md). That way you can iterate on it without having to go through the core Homebrew maintainers for each change. And we can pull it in to core Homebrew once it hits that 30 day mark.

If you set up your own tap, feel free to @-mention me from it or your main repo issues for feedback on the Homebrew formula for it or for working with `brew`.

File: Formula/git-standup.rb (hunk 1 above)
Comment (apjanke) [inline Formula/git-standup.rb]: Yeah, I'd say close this out, put it in a user-maintained tap instead and hash things out there, and when the 30-day marker is passed and things are a bit more stable, you can re-submit it to core Homebrew.

**Comment by md5** [comment]
Thanks @apjanke!

**Comment by mikemcquaid** [comment]
Sorry to contradict you @apjanke but unless you object I think I'm happy to pull this as-is (it now has >1000 stars).

**Comment by md5** [comment]
@mikemcquaid Thanks! I've informed the `git-standup` maintainer to hold off on making a tap for the time being.

**Comment by apjanke** [comment]
Hard to argue with that many stars. No objection.

**Comment by dunn** [comment]
Are we just waiting on kamranahmedse/git-standup#9 now?

**Comment by md5** [comment]
@dunn My understanding from @mikemcquaid was that this PR would be merged as-is. A follow-up PR can be made when @kamranahmedse adds a `man` page and releases a new version with an updated `Makefile` to support installing everything via `make install`.

File: Formula/git-standup.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  test do
+    system "git", "standup", "--help"
```
Comment (mikemcquaid) [inline Formula/git-standup.rb]: This should be `system "#{bin}/git" ...` for future reference (but I'll fix it up locally)

Comment (mikemcquaid) [inline Formula/git-standup.rb]: Ignore me, I'm wrong.

**Pull request merged**